src/Xrefcheck/CLI.hs

```diff
@@ -150,4 +152,32 @@ getCommand = do
         info (helper <*> versionOption <*> totalParser) $
         fullDesc <>
         progDesc "Cross-references verifier for markdown documentation in \
-                 \Git repositories."
+                 \Git repositories." <>
+        (footerDoc $ pure ignoreModesMsg)
+
+ignoreModesMsg :: Doc
+ignoreModesMsg = text $ header <> body
+    where
+        header = "To ignore a link in your markdown, \
+                 \include \"<!-- xrefcheck: ignore mode -->\"\n\
+                 \comment with one of these modes:\n"
+        body = displayS (renderPretty pageParam pageWidth doc) ""
+
+        pageWidth = 80
+        pageParam = 1
+        doc = fillSep $ map formatDesc modeDescr
+
+        modeDescr =
+            [ ("  \"link\"",      L.words $ "Ignore the following link after the comment. \
+                                            \If there is no link, then throw an error.")
+            , ("  \"paragraph\"", L.words $ "Ignore the whole paragraph after the comment.")
+            , ("  \"file\"",      L.words $ "This mode can only be used at the top of markdown \
+                                            \or right after the license. Otherwise throw an error.")
```

Comment:
I would not mention throwing error, this seems not important for me since we said having an "ignore" command at the top is a must.

Also, the practice of using licenses is in fact not wide-spread at all and not every reader will understand why we mention them, I would rather say something like "or right after comments at the top, for instance in case the file has to start from a license note."